src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -103,11 +103,8 @@ public void titlePage() {
         }
 
         // variable project start date and end date
-        if (project.getStart() != null)
-            addReplacement(replacements, "[startdate]", formatter.format(project.getStart()));
-        if (project.getEnd() != null)
-            addReplacement(replacements, "[enddate]", formatter.format(project.getEnd()));
-
+        addReplacement(replacements, "[startdate]", project.getStart() == null ? null : formatter.format(project.getStart()));
+        addReplacement(replacements, "[enddate]", project.getStart() == null ? null : formatter.format(project.getEnd()));
```

Comment:
Would it make sense to be more specific about the value and set it to "" explicitly? `addReplacement` implementation might change in the future, so we are sure what will be inserted.

Reply:
Yes very true, good suggestion.